js/ui/select_box.js

```diff
@@ -714,6 +714,12 @@ const SelectBox = DropDownList.inherit({
 
     _customItemAddedHandler: function(e) {
         const searchValue = this._searchValue();
+
+        if(!searchValue) {
```

Comment:
Why do we write it? Where is an approproate test?

Reply:
https://github.com/DevExpress/DevExtreme/pull/22772/commits/9a43eefb29c548aa868bae1779fe52c7bf31eed1